<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CpuFlagsWidget</class>
 <widget class="QWidget" name="CpuFlagsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>146</width>
    <height>210</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Flags</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QCheckBox" name="zero_checkBox">
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="text">
         <string>Zero</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="subtraction_checkBox">
        <property name="text">
         <string>Subtraction</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="half_carry_checkBox">
        <property name="text">
         <string>Half Carry</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="carry_checkBox">
        <property name="text">
         <string>Carry</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="interrupt_enabled_checkBox">
        <property name="text">
         <string>Interrupt Enabled</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
